src/endpoint/s3/ops/s3_get_bucket.js

```diff
@@ -84,6 +84,12 @@ async function get_bucket(req) {
     };
 }
 
+function get_object_owner(params) {
+    return Object.freeze({
+        DisplayName: params.bucket,
```

Comment:
Hey @vh05 :)
Is this a fix for https://issues.redhat.com/browse/RHSTOR-5501 ? if so can you please attach it to the PR's description?
Originally @sagihirshfeld opened this RFE in order to get the object owner on NC environemnt, but according to https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-ownership-retrieving.html
There are 2 options for object owner - 
1. Bucket owner (default, ACL disabled)
2. Object writer (ACLs enabled)

Since ACLs are not supported - 
1. We should add a comment here about it/add it to the ACLs gap.
2. Return the bucket owner and not the bucket name (on containerized env currently obc name = obc owner account name but this won't work on NC buckets)

WDYT?

Reply:
sure.

About account owner. 

If we list the accounts, then the obc owner's name is synonymous with bucket name. Though just use the bucket name itself since it is same as obc owner's name.

```
- access_keys:
  - access_key: 6FiNkA6WTGvMMHlMlkqC
    secret_key: QqkBPoGEQp0GlPJrzJ/E6zOjFafBi4xxsqBH2ArD
  bucket_claim_owner: n1-3d645c14-c014-4eda-a5c2-be5c81de5433
```

Reply:
@vh05 that's true when using containerized NooBaa.
On Non Containerized deployment we don't have OBCs, this bug was originally opened for NC env, so I think you should fix on both deployments :)